subcommands/waves/rollout.go

```diff
@@ -10,25 +15,81 @@ import (
 )
 
 func init() {
-	cmd.AddCommand(&cobra.Command{
-		Use:   "rollout <wave> <group>",
-		Short: "Rollout a given wave to devices in a given device group",
-		Long: `Rollout a given wave to devices in a given device group.
-Upon rollout a wave becomes available as an update source for production devices in a specific
-device group.  An rollout is not instant, but rather each device in a given group will update to
-this wave's targets at some point in time which is determined by many factors: most important being
-network conditions between a device and update servers, as well as a device update schedule.`,
+	rollout := &cobra.Command{
+		Use:   "rollout <wave> -g <group>",
+		Short: "Roll out a wave to a subset of devices",
+		Long: `Roll out a wave to a subset of devices in a factory (in a device group) matching a wave's tag.
+Upon rollout a wave becomes available as an update source for production devices (in a given device group).
+A rollout is not instant, rather each device will update to the wave's targets at some point in time.
+The exact update time is determined by many factors:
+e.g. device up and down lifecycle, its update schedule, networking between a device and update servers, etc.
+At least one of command arguments is required to limit the subset of devices to roll out to.
```

Comment:
```suggestion
At least one command argument is required to limit the subset of devices to roll out to.
```

Reply:
fixed in force-push

Also renamed `argument -> flag`, as that is the word used in an auto-generated help message.